airflow/providers/apache/druid/hooks/druid.py

```diff
@@ -164,6 +164,10 @@ class DruidDbApiHook(DbApiHook):
     hook_name = "Druid"
     supports_autocommit = False
 
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self.context: dict = kwargs.pop("context", {})
```

Comment:
Better specify `context` it in `__init__` method and add it into the docstring.
This gives more information for endusers about druid context and what the different with airflow context

Reply:
Makes sense, updated.